src/task/methods/scape/config.vsh.yaml

```
@@ -0,0 +1,41 @@
+__merge__: ../../api/comp_method.yaml
+functionality:
+  name: scape
+  info:
+    label: ScAPE
+    summary: judge_2
+    description: ScAPE is a package implementing the neural network model used in the Open Problems – Single-Cell Perturbations challenge hosted by Kaggle. This is the model we used to generate the submission that achieved top <2% performance (16th position out of 1097 teams) in the final leaderboard.
+    reference: pablormier2023scape
+    documentation_url: https://github.com/scapeML/scape
+    repository_url: https://github.com/scapeML/scape
+  arguments:
+    - type: file
+      name: --output_dir
+      direction: output
+      description: Additional output directory for the model.
+      required: true
+      example: path/to/output_dir
+    - type: string
+      name: --cell
+      description: Pre-defined cell type held for pre-training.
+      required: false
+      example: NK cells
+      default: NK cells
```

The unit test runs out of memory on the CI. Could you add one or more arguments to the component with a `default` and a `test_default`? The default will be what gets used during the actual benchmark, while `test_default` is used during CI testing. For example:

```suggestion
      default: NK cells
    - type: integer
      name: --epochs
      description: ...Add description here...
      default: 300
      info:
        test_default: 2
    - type: integer
      name: --epochs_enhanced
      description: ...Add description here...
      default: 300
      info:
        test_default: 2